### PR TITLE
Added clickable phone number on navbar and footer

### LINKED ISSUE #18928: ps_contactinfo module improvements
**New description**

I'd like to extend the original issue, the original issue title is flexible enough to make this seem possible.

**Is your feature request related to a problem?**
- The phone number provided to front page by module ps_contactinfo is not clickable. Using smartphone it is comfortable that phone number is active, the same as email address.
- The eMail Address should be encoded to be protected against SPAM bots

**Describe the solution you'd like**
- enclose the phone number in <a> tags
- For eMail Address make use of Smarty mailto & encode function for better spam bot protection (see #9180)

**Alternatives you've considered**
none

**Additional context**
none

**Original description :**
The phone number provided to front page by module ps_contactinfo is not clickable. 
Using smartphone it is comfortable that phone numer is active, the same as email address.

### PR DESCRIPTION
**Pull request by henri9813**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | This PR permit to make the Phone number clickable, usefull on macbook
| Type?             | improvement|
| Category?         | FO
| BC breaks?        | no
| Deprecations?     |  no
| Fixed ticket?     |  Fixes #18928
| How to test?      | Just setup a prestashop.
| Possible impacts? | No impact.

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/25067)

### CODE REVIEW

File: themes/classic/modules/ps_contactinfo/nav.tpl
Comment (matthieu-rolland) [inline themes/classic/modules/ps_contactinfo/nav.tpl]: it's usually a good practice to put `tel:` before the number:

```suggestion
          '[1]' => "<a href='tel:{$contact_infos['phone']}'>",
```

(except if `$contact_infos['phone']` is already formatted like this)

**Review by matthieu-rolland — COMMENTED** [review]
thank you for your contribution!

we should wait for PM's approval before validating this pull request, meanwhile, I have a small feedback 👍

**Comment by Hlavtox** [comment]
@matthieu-rolland @henri9813 You must remove possible spaces from the phone number, or the link **WILL NOT WORK**. 😉

File: themes/classic/modules/ps_contactinfo/nav.tpl
Comment (henri9813) [reply themes/classic/modules/ps_contactinfo/nav.tpl]: Fixed !!! i forgot that

**Comment by henri9813** [comment]
Done @Hlavtox

File: themes/classic/modules/ps_contactinfo/nav.tpl
Comment (Progi1984) [inline themes/classic/modules/ps_contactinfo/nav.tpl]:
```suggestion
          '[1]' => '<a href="tel:{$contact_infos['phone']|replace:' ':''}">',
```

File: themes/classic/modules/ps_contactinfo/ps_contactinfo.tpl
Comment (Progi1984) [inline themes/classic/modules/ps_contactinfo/ps_contactinfo.tpl:45]:
```suggestion
        '[1]' => '<a href="tel:{$contact_infos['phone']|replace:' ':''}">',
```

**Review by Progi1984 — COMMENTED** [review]
Small quote feedbacks

**Comment by henri9813** [comment]
Hello,

I apply your suggestions

**Comment by henri9813** [comment]
Done @Progi1984

**Review by kpodemski — DISMISSED** [review]

**Review by Progi1984 — DISMISSED** [review]

**Review by NeOMakinG — DISMISSED** [review]
Cool!

**Comment by NeOMakinG** [comment]
@henri9813 I restarted the CI but looks like something is wrong : 
image

**Comment by henri9813** [comment]
Hello,

I know, it's the syntax wanteed by @Progi1984  :/ 

Should i revert ?

File: themes/classic/modules/ps_contactinfo/nav.tpl
Comment (NeOMakinG) [reply themes/classic/modules/ps_contactinfo/nav.tpl]: Injecing {$any} vars is not possible on smarty, you should use double quotes such as "

**Comment by NeOMakinG** [comment]
@henri9813 well, sorry for this mistake but injecting vars inside single quotes is impossible with smarty, you should revert it... 😥 

Edit: How well, it's not var injecting there, but only a syntax mistake, does it work with " on phone key?

**Comment by Julievrz** [comment]
> This pull request seems to contain new translation strings. I have summarized them below to ease up review:
> 
> * `messages`
>   
>   * [ ]  [`Call us: [1]%phone%[/1]`](https://github.com/PrestaShop/PrestaShop/pull/25067/files#diff-7090d8a84ce18cb82df3780616a7df2ee0d288f69f4f764f47ce9f10a80bceacR32)
> 
> (Note: this is an automated message, but answering it will reach a real human)

~~Hi @henri9813, this string should not be in the "messages" domain. It should be in Modules.Contactinfo.Shop~~

~~Thanks in advance :-)~~

**Comment by henri9813** [comment]
Hello,

Fixed.

@Julievrz  i didn't change this part, if something is to done with this, it should be in another pr non ?

Best regards

**Review by Progi1984 — APPROVED** [review]

**Comment by Julievrz** [comment]
> Hello,
> 
> Fixed.
> 
> @Julievrz i didn't change this part, if something is to done with this, it should be in another pr non ?
> 
> Best regards

Hi, my bad, you're right. ;-) I approve the wording then.

**Comment by henri9813** [comment]
@NeOMakinG can you approve please ?

**Review by NeOMakinG — APPROVED** [review]

**Comment by henri9813** [comment]
Someone can merge this please ?

**Comment by Hlavtox** [comment]
@henri9813 Wait a bit, **Waiting for QA** means that somebody from QA team must check the PR out, if everything works fine and it does not break anything. It will be merged right after that. :-)

**Comment by henri9813** [comment]
Thanks @Hlavtox

**Comment by florine2623** [comment]
LGTM

Thanks @henri9813 !

**Comment by PierreRambaud** [comment]
Thank you @henri9813

**Pull request merged**
